### ISSUE TITLE
Provide documentation 

### ISSUE THREAD

**Issue by chucklecode** [kochlisGit/ProphitBet-Soccer-Bets-Predictor#88]
Hey bro, I recently stumbled upon this your software, the prophit-bet software, I cloned it and decided to run the code, but I'm having problems like in the GUI you did not provide enough documentation about training models, again, most parameters in the new season like the HW and so on are saying 'nan', still on the documentation part again, I find it difficult to predict future games as there are not listed in the GUI. Please can you help explain these problems for me or provide a well documented Documentation. Thanks.

**Comment by lionfish56** [comment]
Yes I‘ve seen this problem too. At Jupiter League, German Bundesliga and more. If kochlisGit could give us a hint to solute this, it would be great. I thought the problem is the year 2021 in creating the league and change it to 2022, but I don‘t think so. The Brazilian Serie A or China Superleague are in their 2024 league and there are no problems with it.
